src/KokkosComm.hpp

```diff
@@ -28,8 +29,49 @@
 
 #include <Kokkos_Core.hpp>
 
+#include <algorithm>
+#include <cstdio>
+#include <string_view>
+
 namespace KokkosComm {
 
+inline void initialize(int &argc, char *argv[], int mpi_required_thread_lvl) {
```

Comment:
```suggestion
inline void initialize(int &argc, char *argv[], int mpi_required_thread_lvl = MPI_THREAD_SINGLE) {
```
Providing the default argument would make it equivalent to `MPI_Init` when called without last argument.

Reply:
Adding this with the `MPI_THREAD_*` wrapper suggested by Joseph, thank you :+1: